data-raw/create_adams_data.R

```diff
@@ -0,0 +1,167 @@
+# ensure every packages are installed
+
+update_pkg <- FALSE
+ignore_templates <- list(
+  "admiralonco" = c("ad_adrs_basic.R")
+)
+
+save_rda <- function(data, file_path, new_name) {
+  if (missing(new_name)) {
+    save(data, file = file_path, compress = "bzip2")
+  } else {
+    assign(new_name, data)
+    save(list = new_name, file = file_path, compress = "bzip2")
+  }
+}
+
+load_rda <- function(fileName) {
+  load(fileName)
+  get(ls()[ls() != "fileName"])
+}
+
+write_doc <- function(data, dataset_name) {
+  # create documentation for the current dataset
+  # TODO: use metatools/metacore for doc  ?
+  doc_string <- paste(
+    sprintf("#' Dataset %s", dataset_name),
+    "#'",
+    sprintf("#' %s dataset", dataset_name),
+    "#'",
+    sprintf("#' @name %s", dataset_name),
+    "#' @docType data",
+    sprintf("#' @format A data frame with %s columns:", ncol(data)),
+    "#'   \\describe{",
+    paste(sapply(names(data), function(col_name) {
+      paste(sprintf("#'     \\item{ %s }{%s}", col_name, col_name))
+    }, USE.NAMES = FALSE), collapse = "\n"),
+    "#'   }",
+    "#'", sprintf("#' @source Generated from create_adams_data.R script."),
```

Comment:
I would replace `create_adams_data.R` with the name of the template file and the package. I think this is more helpful for the users.

Reply:
I think this is a good idea ! But maybe I would create a separated issue for this (I think @manciniedoardo and @kaz462 want to release pharmaverseadam early January)